k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -129,8 +130,12 @@
 public class MessagingController {
     public static final long INVALID_MESSAGE_ID = -1;
 
-    private static final Set<Flag> SYNC_FLAGS = EnumSet.of(Flag.SEEN, Flag.FLAGGED, Flag.ANSWERED, Flag.FORWARDED);
-
+    private static final Set<Flag> PREDEFINED_SYNC_FLAGS =
```

Comment:
This feels like it belongs in the Flags / FlagsManager class.

Reply:
Made it `FlagManager.getSyncFlags()`.